chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/ComplaintDAO.java

```diff
@@ -11,7 +11,7 @@
 
     List<ComplaintStatusTypeDTO> getComplaintStatusTypes();
 
-    List<ComplaintDTO> getAllComplaints();
+    List<ComplaintDTO> getAllComplaints() throws EntityRetrievalException;
```

Comment:
Just wondering how/why this might throw this particular exception. If it's valid that's fine, I'm just thinking that I personally have mostly used it for like looking something up by ID but that ID doesn't exist in the database.